.github/workflows/docs-build.yml

```diff
@@ -35,6 +35,7 @@ env:
 
 jobs:
   docs-checks:
+    if: github.event.pull_request.draft == false
```

Comment:
I find building docs on drafts useful

Reply:
The HTML build is still there, this is just for doctests and linkcheck